src/lib.rs

```diff
@@ -131,7 +129,7 @@ impl Encode for Method {
 
         if let Some(constraints) = &self.constraints {
             #[allow(clippy::cast_possible_truncation)]
```

Comment:
Do we still need this attribute? looks a bit weird inline

Reply:
Prefer leave it for now, its for the lossy `as` cast